docs/pages/server-access/guides/ssh-pam.mdx

```diff
@@ -177,6 +177,12 @@ $ cat /etc/motd
 
 ## Create local Unix users on login
 
+<Admonition type="warning">
+This guide is obsolete and will be removed in a future version. Refer to the
```

Comment:
```suggestion
Using PAM to create local users on login is obsolete and will be removed in a future version. Refer to the
```
Suggested to be more specific. Otherwise, I was wondering why this admonition came so late in an obsolete guide.